ts/utils/paymentMethod.ts

```diff
@@ -60,33 +57,24 @@ export const getPaymentMethodHash = (
       return pm.info.uidHash;
     case "Bancomat":
     case "CreditCard":
-    case "Privative":
       return pm.info.hashPan;
+    case "Privative":
+      return undefined;
```

Comment:
This method was not returning undefined so it should no return undefined.
Instead, remove `Privative` from the `RawPaymentMethod` kind